app/helpers/feeds_helper.rb

```diff
@@ -68,4 +68,8 @@ def feed_retry_image_path(feed, form)
       podcast_feed_path feed.podcast, feed, uploads_retry_params(form)
     end
   end
+
+  def apple_feed?(feed)
+    feed.type == "Feeds::AppleSubscription"
```

Comment:
Just noting I added a `feed.apple?` method to the model in #1019, to do the same thing.  After these both merge, we should probably consolidate them.